defaults/preferences/nostalgy.js

```diff
@@ -1,2 +1,2 @@
 // See http://kb.mozillazine.org/Localize_extension_descriptions
-pref("extensions.nostalgy@alain.frisch.description", "chrome://nostalgy/locale/nostalgy.properties");
+//pref("extensions.nostalgy@alain.frisch.description", "chrome://nostalgy/locale/nostalgy.properties");
```

Comment:
Cannot the file be dropped entirely?

Reply:
no - I tried that already, because I don't like two files with same name and differiing content. But something breaks if it is removed.

Reply:
interesting, in my unzip of old 0.2.36 xpi, this was already commented